data/json/mapgen/private_resort.json

```diff
@@ -0,0 +1,1841 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "p_resort_1nw" ],
+    "weight": 1000,
+    "object": {
+      "fill_ter": "t_metal_floor",
+      "rows": [
+        "....______..______......",
+        ".FFW======WW======WFFFFF",
+        ".F.%:____:__:____:%.....",
+        ".F..:____:__:____:......",
+        ".F..:____:__:____:____..",
+        ".F.g:____:__:____:____..",
+        ".F.g:____:__:____:____..",
+        ".F.g:____:__:____:____..",
+        ".F.g:____:__:____:____..",
+        ".F..:____:__:____:......",
+        ".F..W=====WW=====WWW|||W",
+        ".F..Wmmmmm%mmmmmm%Wfmmmf",
+        ".F..WmmmmmmmmmmmmmWfmmmf",
+        ".F..WmmZmmmZmmmmmmWfmmmf",
+        ".F..WmZmmmmmmmmmmmWfmmmf",
+        ".F..WmZmmmZmmmmmmmWfmmmf",
+        ".F..WmmmmZmmmmmmmmWW|||W",
+        ".F..WmmmmZZmmmmmmm|mmmmm",
+        ".F..WmmmmZZm>>mmmm|mmmm<",
+        ".F..W|W|WWWWWWWWWWWmZmmL",
+        ".F..W W,,Bb,,Bb,ggWZZmmL",
+        ".F..W W,,,,,,,?,,,|mmmmL",
+        ".F..W W,,?,,,,,,,,|mmmmL",
+        ".F..W W,,Bb,,Bb?,,WmmmmL"
+      ],
+      "palettes": [ "p_resort_palette_main_floor" ],
+      "place_signs": [
+        { "signage": "Private property. No trepassing!", "x": 4, "y": 0 },
+        { "signage": "Private property. No trepassing!", "x": 18, "y": 0 }
+      ],
+      "place_items": [
```

Comment:
In general, place_items is the most difficult way to do item spawns and we try not to use it when possible. It's going to be exceptionally hard to review item spawns in here with this method.

I will go through it though. At a glance, I'd say you probably want to take a look at some of the SUS item groups and consider using a few of those in the kitchens at least. in general your item spawn rates seem low for what you're going for here but I won't know until I've figured out what they're all referring to.

Reply:
So far spanws appeared sane and reasonable. SUS seem very practical too if that'll be required after all.